### PR TITLE
Add Excel test again

### PR DESCRIPTION
**Pull request by zaucker**
- Enable temporarily disabled test
- Add pdf route test

### CODE REVIEW

**Comment by zaucker** [comment]
It seems the failing `get_excel_export` test in `t/routes-application.t` might be `Cro` problem. It works with version 0.8.3 and fails with 0.8.4.

**Comment by jnthn** [comment]
@zaucker The "Cro problem", assuming it's the same thing I investigated before, turned out to be because the test was wrong but we got away with it because of a now-fixed bug (it didn't validate form-encoded bodies properly before 0.8.4).

File: t/routes-application.t [hunk 1]
```diff
@@ -1,4 +1,3 @@
-use v6;
```
Comment (jnthn) [inline t/routes-application.t:1]: Any reason to remove this, given we recently added it everywhere?

File: t/routes-application.t [hunk 2]
```diff
@@ -42,6 +41,8 @@ my $fake-store = mocked(Agrammon::Web::Service,
         get-output-variables => -> $user, $dataset-name {
             %( :variable('x'), :value(2) )
         },
+        get-pdf-export => -> $user, %params {
+        },
```
Comment (jnthn) [inline t/routes-application.t:45]: If you're not adding any custom result or behavior to the method, and only want to record its being called, you needn't specify an override for it in the mock.

File: t/routes-application.t [hunk 3]
```diff
@@ -115,23 +116,37 @@ subtest 'Get output variables' => {
...
+subtest 'Get excel export' => {
+    test-service routes($fake-store), :$fake-auth, {
+        test-given '/export/excel', {
+            test post(
+                    content-type => 'application/x-www-form-urlencoded',
+                    body => {
+                        :datasetName('TestSingle'),
+                        :language('de'),
+                    }),
+                    status => 200;
```
Comment (jnthn) [inline t/routes-application.t:128]: Maybe worth testing the content type of the response too?

File: t/routes-application.t (hunk 1 above)
Comment (zaucker) [reply t/routes-application.t:1]: Branch from the past ... added again.

File: t/routes-application.t (hunk 2 above)
Comment (zaucker) [reply t/routes-application.t:45]: Ah, didn't know. Removed for the moment.

File: t/routes-application.t (hunk 3 above)
Comment (zaucker) [reply t/routes-application.t:128]: How would I do that? And doesn't OpenAPI enforce that anyway?

Comment (jnthn) [reply t/routes-application.t:128]: > How would I do that?

Pass `content-type => '...'` after `status => 200`. [Docs](https://cro.services/docs/reference/cro-http-test#content-type). 

> And doesn't OpenAPI enforce that anyway?

Good point, yes, so it is a bit surplus to requirements. So let's not worry about it.

**Review by jnthn — APPROVED** [review]

**Pull request merged**